homepage/templates/faq.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+
+{% block title %}FAQ{% endblock title %}
+
+{% block content %}
+<section id="features" class="features section custom-reduce-padding-2">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 col-sm-12 col-xs-12">
+                <h1 class="title">FAQ</h1>
+                <div class="content">
+                    <h2>I am having trouble installing Orange.</h2>
+                    <p>Make sure you are downloading the latest version. You can also try to download a standalone installer or Anaconda distribution.</p>
+                    <p>Still no luck? Check our <a href="https://github.com/biolab/orange3/issues">issue tracker</a> for similar issues or report a new one.</p>
+
+                    <h2>I am having trouble installing an add-on.</h2>
+                    <p>Make sure you are using the latest version of Orange. Alternatively, you can try with Anaconda distribution and install the add-on in the terminal. You can use pip or conda to install a package directly. E.g.:
+                    <pre>pip install orange3-text<br>conda install orange3-timeseries</pre></p>
+
+                    <h2>I get an error when using Orange.</h2>
+                    <p>First, make sure you are using the latest version of Orange. If this doesn’t fix the issue, you should be able to see an error window pop up, prompting you to submit the issue to the developers. Please use this option as it allows us to recreate the issue and fix it. If this is not possible, please report the issue to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>.</p>
+
+                    <h2>I believe there’s a bug in Orange.</h2>
+                    <p>First, make sure you are using the latest version of Orange. If the bug is still present, please submit an issue to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>. Follow the template and describe the issue as well as possible. Tag it with <b>bug</b> tag.</p>
+
+                    <p>Before submitting a new issue, check if someoone has already reported the same problem. Upvote the issue with a +1 and/or add your comment. Only consider opening a new issue if you cannot find a corresponding existing issue.</p>
+
+                    <h2>There’s a feature I wish to see in Orange.</h2>
+                    <p>Please submit a feature request to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>. Describe what you wish to see in Orange and, if possible, provide a visual example. Tag the request with <b>enhancement</b>. If enough people seem to be interested in the request, we will try to implement it.</p>
```

Comment:
I would prefer "Submit a PR"/"Create and add-on" before "Open an issue". I am not sure whether people without commit rights can tag issues.

Reply:
```
If enough people seem to be interested in the request, we will try to implement it.
```
I am not sure we want to promote this.

Reply:
I think we don't want to promote it.
The answer here is probably along the lines of "we have limited resources...", "happy to receive a PR"

Reply:
I'm just gonna delete this, even though I think we should say something like 'If enough people seem to be interested in the request, we will try to implement it (resources permitting).' or (should our limited resources allow it). Not promising, but not rejecting either.